src/Ledger/Prelude.agda

```diff
@@ -20,6 +20,7 @@ open import Interface.DecEq public
 open import Ledger.Interface.HasCoin public
 open import Interface.HasAdd public
 open import Interface.HasAdd.Instance public
+open import Interface.HasRawPartialOrder public
```

Comment:
The instance should also be exported from here

Reply:
That's how I did it originally, but it resulted in a name clash/ambiguity.  But okay, I'll see if I can put the instance back here and resolve the name clash.